stream/client/async_client.py

```diff
@@ -0,0 +1,276 @@
+import logging
+
+import aiohttp
+from aiohttp import ClientConnectionError
+
+from stream import serializer
+from stream.client.base import BaseStreamClient
+from stream.collections import AsyncCollections
+from stream.feed.feeds import AsyncFeed
+from stream.personalization import AsyncPersonalization
+from stream.reactions import AsyncReactions
+from stream.serializer import _datetime_encoder
+from stream.users import AsyncUsers
+from stream.utils import (
+    get_reaction_params,
+    validate_feed_slug,
+    validate_foreign_id_time,
+    validate_user_id,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class AsyncStreamClient(BaseStreamClient):
+    def __init__(
+        self,
+        api_key,
+        api_secret,
+        app_id,
+        version="v1.0",
+        timeout=6.0,
+        base_url=None,
+        location=None,
+    ):
+        super().__init__(
+            api_key,
+            api_secret,
+            app_id,
+            version=version,
+            timeout=timeout,
+            base_url=base_url,
+            location=location,
+        )
+        self.session = aiohttp.ClientSession
```

Comment:
I don't think we need `session` variable at all. Why not just:

```py
        async with aiohttp.ClientSession() as session:
            async with session.request(
```

Reply:
ok